src/clj/rems/workflow/dynamic.clj

```diff
@@ -694,7 +789,7 @@
       (invitation-token-error application (:token cmd))
       {:success true
        :result {:event/type :application.event/member-joined
-                :event/time (:time cmd)
+                :event/time (:time cmd) ;; TODO these are redundant
                 :event/actor (:actor cmd)
                 :application/id (:application-id cmd)
```

Comment:
The `ok` function can be used to get rid of the `:success true` and the time, actor and application-id can be removed because `handle-command` will already fill them in by calling `enrich-result`.